package.json

```diff
@@ -32,6 +36,18 @@
   },
   "homepage": "https://github.com/bloomberg/coffeechat#readme",
   "devDependencies": {
+    "@storybook/addon-a11y": "^6.2.9",
+    "@storybook/addon-actions": "^6.2.9",
+    "@storybook/addon-controls": "^6.2.9",
+    "@storybook/addon-docs": "^6.2.9",
+    "@storybook/addon-storyshots": "^6.2.9",
+    "@storybook/addon-storysource": "^6.2.9",
+    "@storybook/addon-viewport": "^6.2.9",
+    "@storybook/addons": "^6.2.9",
+    "@storybook/client-api": "^6.2.9",
+    "@storybook/react": "^6.2.9",
+    "@storybook/storybook-deployer": "^2.8.7",
+    "@storybook/theming": "^6.2.9",
```

Comment:
I was reading Storybook 6 blog when I came across this new feature https://github.com/storybookjs/testing-react 
...seems useful to help reduce efforts while writing tests.should we add this too?

also, just fyi, storybook 6 doesn't support npm 7's peerDependency but since we are anyway not planning to publish this as a package..I guess it doesn't matter much.